sdk/servicebus/azure-servicebus/tests/test_sessions.py

```diff
@@ -487,7 +487,7 @@ def test_session_by_servicebus_client_browse_messages_client(self, servicebus_na
             with sb_client.get_queue_session_receiver(servicebus_queue.name, session_id=session_id) as receiver:
                 messages = receiver.peek_messages(5)
                 assert len(messages) == 5
-                assert all(isinstance(m, PeekedMessage) for m in messages)
+                assert all(isinstance(m, ServiceBusPeekedMessage) for m in messages)
```

Comment:
I'm guessing we remove this type in a subsequent PR?

Reply:
yes, it would go into the settlement movement PR